types/react/index.d.ts

```diff
@@ -2034,7 +2034,7 @@ declare namespace React {
     interface IframeHTMLAttributes<T> extends HTMLAttributes<T> {
         allow?: string;
         allowFullScreen?: boolean;
-        allowTransparency?: boolean;
+        allowtransparency?: boolean;
```

Comment:
I think we just want to remove that attribute. It isn't a standard attribute as far as I can tell (https://html.spec.whatwg.org/multipage/iframe-embed-object.html#the-iframe-element). We don't add types for non-standard attributes.

Reply:
I think its should be another PR

Reply:
We don't include lowercase attributes either. Only their supported camel-cased versions. Neither applies here. I can open a separate on that removes it. But then this PR won't be accepted. Your choice.

Reply:
Open a separate PR, please

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/48605